### ISSUE TITLE
Next.js app dir

### ISSUE THREAD

**Issue by oscartbeaumont** [oscartbeaumont/rspc#194]
- [ ] Example of using rspc with the Next.js app dir.
 - [ ] Also lazily load the Tauri transport so it can be used in Next.js without causing issues.

**Comment by dukeeagle** [comment]
Would love to see your crack at this! I solved the Tauri-Next.js issue by just writing a custom `TauriTransport`

**Comment by oscartbeaumont** [comment]
Yeah it should be fairly easy to fix but as I don't use that stack I haven't been able to prioritize it.
